src/components/MovieCard/MovieCard.scss

```diff
@@ -1 +1,4 @@
-// 
+.card-image img {
+  object-fit: contain;
+  margin-top: 10px; // now it looks better
```

Comment:
Delete the comment.

Reply:
Hi Daria=)
Fixed according to your comments, thank you!